C/manual/ch_Finance-Quote.xml

```diff
@@ -288,7 +284,30 @@ SYNOPSIS
     <sect2 id="fq-check-version">
       <title>Displaying the Finance::Quote Version and Supported Sources</title>
 
-      <para><command>gnucash-cli --quotes info</command> produces the following output:
+      <abstract>
+        <para>The <replaceable>--quotes info</replaceable> mode returns the version number of the currently
+          installed &app-fq; module and a list of available sources. It informs you also if there is
+          a problem with your installation and gives the reasons.
```

Comment:
s/mode/option/. Mode in this context would mean a persistent state in which one may perform subsequent actions or the program behaves a certain way. An option is a one-time modifier. `gnucash-cli --quotes info` displays the F::Q version and the sources it provides without affecting future invocations of `gnucash-cli` so it's an option, not a mode.